From 1eaba181577de206d3d43ec7886d0353fc0c9f2a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=2E=20David=20Ib=C3=A1=C3=B1ez?= <jdavid.ibp@gmail.com>
Date: Mon, 17 Sep 2018 20:19:41 +0200
Subject: [PATCH] Do not require older pycparser<2.18

Fixes #819
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index d1fc4e90..21d9276e 100644
--- a/setup.py
+++ b/setup.py
@@ -195,7 +195,7 @@ def run(self):
       long_description=long_description,
       packages=['pygit2'],
       package_data={'pygit2': ['decl.h']},
-      setup_requires=['cffi', 'pycparser<2.18'],
+      setup_requires=['cffi'],
       install_requires=['cffi', 'six'],
       zip_safe=False,
       cmdclass=cmdclass,
