--- pygit2-0.21.3/docs/conf.py.orig	2014-09-18 16:25:58.336102616 +0200
+++ pygit2-0.21.3/docs/conf.py	2014-09-18 16:30:38.056090370 +0200
@@ -18,7 +18,8 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 version_string = sys.platform.rstrip(digits) + "-" + os.uname()[4] + "-" + ".".join(platform.python_version_tuple()[0:2])
-sys.path.insert(0, os.path.abspath('../build/lib.' + version_string))
+builddir = os.getenv('PACKAGE_BUILD', '../build')
+sys.path.insert(0, os.path.abspath(builddir + '/lib.' + version_string))
 
 # -- General configuration -----------------------------------------------------
 
